README.md

```diff
@@ -152,6 +155,65 @@ For example, to use the GNOME gedit flatpak, use:
 ["flatpak", "run", "--filesystem=xdg-run/textern", "org.gnome.gedit"]
 ```
 
+##### Flatpak Firefox
```

Comment:
I think this should be at the same level as "Terminal editors" and "GUI Editors" instead of being inside the latter, so just

```suggestion
### Flatpak Firefox
```

?